kong/plugins/zipkin/opentracing.lua

```diff
@@ -194,7 +194,7 @@ function OpenTracingHandler:log(conf)
 		opentracing.body_filter_span:finish(ctx.KONG_BODY_FILTER_ENDED_AT / 1000)
 	end
 
-	request_span:set_tag("http.status_code", ngx.status)
+	request_span:set_tag("http.status_code", string.format("%d", ngx.status))
```

Comment:
`http.status_code` is also meant to be an integer (See https://github.com/opentracing/specification/blob/master/semantic_conventions.md)